autospotting.go

```diff
@@ -98,6 +105,9 @@ func (c *cfgData) parseCommandLineFlags() {
 			autospotting.OnDemandPercentageLong+
 			"\n\tIt is ignored if min_on_demand_number is also set.")
 
+	flag.BoolVar(&c.KeepInstanceType, "keep_instance_type", false,
+		"If specified, the spot instances will have the same instance type "+
+			"as the initial on-demand instances")
```

Comment:
Why not making it a string, so that any type can be given?
A special type such as `current` could be given to indicate to take the same type as the one in the ASGs, what do you think?